app/javascript/readingList/readingList.jsx

```diff
@@ -243,6 +272,7 @@ export class ReadingList extends Component {
             </div>
             <div>
               {items.length > 0 ? itemsToRender : this.renderEmptyItems()}
+              {collectionItemsToRender }
```

Comment:
To get this in its own container I think we can reuse some of the div classes above, so wrap it in like
```
<div className="items-container">
    <div className={`results`}>
            <div className="results-header">
                  Collections
             </div>
             {collectionItemsToRender}
....
```